### DISCUSSION TITLE
Query Related to contact model

### DISCUSSION THREAD

**Discussion by deepakbiitk** [peridigm/peridigm#323] [General]
Hi all,

In the example disk impact. I tried to change the material model of disk material. If I use material model as "Isotropic Hardening Correspondence" the simulation runs fine. However, If I use "Isotropic Hardening Bond Associated Correspondence" it throws an error.
**** Error in DataManagerSynchronizer::checkFieldValidity():  Field Deformation_Gradient_X not present in block block_3

Here block_3 has material model as 'Elastic'

Any suggestions regarding correcting the error. Thanks.

**Comment by johntfoster** [comment]
You need to use the same material model in both blocks.  If you want
block_3 to remain elastic, just set the "Yield Stress" to be a very large
number.

**Reply by deepakbiitk** [reply]
Thank you Dr. Foster for the recommendation.
